apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/journalpost/EndreAvsenderMottaker.tsx

```diff
@@ -25,7 +25,7 @@ export const EndreAvsenderMottaker = ({
   const [rediger, setRediger] = useState(false)
 
   const lagreEndring = (avsenderMottaker: AvsenderMottaker) => {
-    oppdaterAvsenderMottaker(avsenderMottaker)
+    oppdaterAvsenderMottaker({ ...avsenderMottaker, idType: BrukerIdType.FNR })
```

Comment:
```suggestion
    oppdaterAvsenderMottaker({ ...avsenderMottaker, idType: avsenderMottaker.idType ?? BrukerIdType.FNR })
```
evt.? i tilfelle den er satt

Reply:
Ja, tenker vi kun setter FNR hvis idType er null, pg fnr er gyldig fnr.